plugins/plugin-bash-like/src/test/bash-like/pty-copy-paste.ts

```diff
@@ -41,8 +41,6 @@ localDescribe('xterm copy paste', function (this: common.ISuite) {
 
   it(`should echo ${emittedText}`, async () => {
     try {
-      const res = cli.do(`echo ${emittedText}`, this.app)
```

Comment:
Remove the expression will cause the command not executed. Maybe change to ```await cli.do(`echo ${emittedText}`, this.app)```?

Reply:
`res` is not being used anywhere, so I can't see why we need this line.